templates/tutorialv2/includes/meta/image_illu.part.html

```diff
@@ -1,6 +1,6 @@
 {% load thumbnail %}{% spaceless %}
     {% if content.image %}
-        {% if request.GET.twittercard == "large"  %}
+        {% if request.GET.Xcard == "large"  %}
```

Comment:
```suggestion
        {% if request.GET.twittercard == "large"  %}
```

Le nom des paramètres `meta` n'a pas changé : le nom ne serait pas cohérent [avec le nom de la métadonnée qui reste `twitter:card`](https://developer.x.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image). De plus, cela casserait tous les liens avec la grande carte existants (il faudrait _a minima_ accepter les deux noms).